Assets/MixedRealityToolkit/Utilities/Scripts/Billboard.cs

```diff
@@ -30,6 +30,7 @@ public class Billboard : MonoBehaviour
         /// The axis about which the object will rotate.
         /// </summary>
         [Tooltip("Specifies the axis about which the object will rotate.")]
+        [SerializeField]
         public PivotAxis PivotAxis = PivotAxis.XY;
```

Comment:
public fields are already serialized, so this attribute is redundant, unless you plan to also make the field private per coding guidelines.

Reply:
It's public here, but private in the master branch. Looks like there are probably some other changes to be made.

Reply:
Changes from the original Dev -> Master PR have now been migrated.